packages/strapi-generate-new/lib/merge-template.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const path = require('path');
+const fse = require('fs-extra');
+const fetch = require('node-fetch');
+const unzip = require('unzip-stream');
+const _ = require('lodash');
+
+// Specify all the files and directories a template can have
```

Comment:
Just wanted to say thank you for the comments :hugs: I wish more of our code had comments :laughing: